docs/images.md

```diff
@@ -128,6 +179,9 @@ pulled using imagePullSecrets.  That is, imagePullSecrets does *NOT* protect you
 images from being seen by other users in the cluster.  Our intent
 is to fix that.
 
+You can use this in conjunction with a per-node `.dockerfile`.  The credentials
```

Comment:
Testing on all the other providers is impractical and so I'm reluctant to put an explicit positive in for them.